index.html

```diff
@@ -37,154 +36,213 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
-                   <li class="li-lang">Ruby</li>
-                   <li class="li-lang">html</li>
-                   <li class="li-lang">css</li>
-               </ul>
+                <div class="about">
+                </div>
+                <div class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </div>
+                <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+            </div>
+                </div>  
+                
+                <div class="skills-container">
+                <article class="project2">
+                <div class="language-icon"><img src="./images/icon-languages (1).png" alt=""></div>
+                <div class="lang-text">Languages </div>
+                    <ul>
+                    <div class="lang-text-list">
+                        <li  class="li-lang">Javascript</li>
+                        <li class="li-lang">Ruby</li>
+                        <li class="li-lang">html</li>
+                        <li class="li-lang">css</li>
+                    </div>
+                    </ul>
+
+                </article>
+                <article class="project2">
+                    <div class="language-icon"> <img src="./images/icon-frameworks (1).png" alt=""></div>
+                    <div class="lang-text">Frameworks </div>
+                    <div class="">
+                    <ul>
+                    <div class="lang-text-list">
+                        <li class="li-lang">Bootstrap</li>
+                        <li class="li-lang">Ruby on rails</li>
+                        <li class="li-lang">Rspec</li>
+                        <li class="li-lang">css</li>
+                        <li class="li-lang">Capybara</li>
+                        <li class="li-lang">Selenium</li>
+                    </div>
+                    </ul>
+                    </div>
+    
+                </article>
+                <article class="project2">
+                    <div class="language-icon"><img src="./images/icon-skills (1).png" alt=""></div>
+                    <div class="lang-text">Skills</div>
+                    <div class="">
+                    <ul>
+                    <div class="lang-text-list">
+                        <li class="li-lang" >CodeKit</li>
+                        <li class="li-lang">Github</li>
+                        <li class="li-lang">Codepen</li>
+                        <li class="li-lang">Gitlab</li>
+                        <li class="li-lang">Terminal</li>
+                    </div>
+                    </ul>
+                    </div>
+    
+                </article>
+                </div>
+            
+            </section>
+            </section>
```

Comment:
- You have done a great job so far with your desktop design 👏 👏. But there are some little differences between your design and the design in the Figma template you have chosen. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project 👇 (Please pay attention to the marked sections in the picture of the Figma template I attached below):
-- Please add a border radius to the list of technologies 🙏 
-- Please attach a border radius to the `Languages, Skills and Framework` cards 🙏 🙏 
-- The background image on the right is not aligned properly as seen in the Figma template
-- There is a missing `Get My Resume` button between the about me text and the skill cards, please fix this 🙏 🙏 
<img width="958" alt="eshnsfnf" src="https://user-images.githubusercontent.com/73607512/206673224-1ffc3e83-1123-4784-b186-5589af1e2538.PNG">

Figma Template:
<img width="545" alt="rrwrhrh" src="https://user-images.githubusercontent.com/73607512/206673281-991d1f79-b18b-4acf-828b-bb0a512d1d0a.PNG">